vpc.go

```diff
@@ -176,18 +181,21 @@ func (c *VPCCollector) collectSubnetsPerVpcQuota(ch chan<- prometheus.Metric) {
 func (c *VPCCollector) collectSubnetsPerVpcUsage(ch chan<- prometheus.Metric, vpc *ec2.Vpc) {
 	ctx, cancelFunc := context.WithTimeout(context.Background(), c.e.timeout)
 	defer cancelFunc()
-	describeSubnetsOutput, err := c.ec2.DescribeSubnetsWithContext(ctx, &ec2.DescribeSubnetsInput{
-		Filters: []*ec2.Filter{&ec2.Filter{
+	var subnets []*ec2.Subnet
+	err := c.ec2.DescribeSubnetsPagesWithContext(ctx, &ec2.DescribeSubnetsInput{
+		Filters: []*ec2.Filter{{
 			Name:   aws.String("vpc-id"),
 			Values: []*string{vpc.VpcId},
-		}},
+		}}}, func(page *ec2.DescribeSubnetsOutput, lastPage bool) bool {
+		subnets = append(subnets, page.Subnets...)
```

Comment:
Same here, i'd suggest to only save the actual number of subnets, and not the actual subnets so it's easier to understand what we are attempting.